spec/fixtures/cookbooks/oneview_test/metadata.rb

```diff
@@ -2,6 +2,6 @@
 maintainer       'test cookbook'
 license          'All rights reserved'
 description      'A test cookbook for the oneview cookbook'
-version          '0.1.0'
+version          '0.1.1'
```

Comment:
No, we don't. It was just to keep in track with the original cookbook version...